lib/constants/constants.dart

```diff
@@ -320,3 +320,25 @@ class WidgetKeys {
   static const Key mainAppBarLogo = Key("mainAppBarLogo");
   static const Key termsAndConditionsSwitch = Key("termsAndConditionsSwitch");
 }
+
+/// Enum holding all updateable player fields
```

Comment:
Uups this enum is not used anymore. I created it in an intermediate step. I'll delete it in a cleanup commit.